packages/uikit-react-native/src/domain/groupChannelThread/component/GroupChannelThreadHeader.tsx

```diff
@@ -30,21 +37,22 @@ const GroupChannelThreadHeader = ({ onPressHeaderLeft }: GroupChannelThreadProps
   };
 
   return (
-    <HeaderComponent
-      clearTitleMargin
-      title={
-        <View style={styles.titleContainer}>
-          <View style={{ flexShrink: 1 }}>
-            <Text h2 color={colors.onBackground01} numberOfLines={1}>
-              {headerTitle}
-            </Text>
-            {renderSubtitle()}
+    <PressBox onPress={onPressHeader}>
```

Comment:
Is it expected to navigate back when the header title area is pressed, instead of using the back button? What behavior was the QA team expecting?

Reply:
The QA team expects navigation to go back when either the back button or the header title is pressed. Also, the behavior on other platforms is consistent with that the QA team expects.

Reply:
Thank you for the explanation. Do you happen to know the reason why? 🤔

Reply:
I understood the behavior incorrectly.
When the back button is pressed, it navigates back. When the subtitle is pressed, it navigates to the parent message.
I will fix the issue. @bang9

Reply:
[I have fixed](https://github.com/sendbird/sendbird-uikit-react-native/pull/182/commits/484199d59e0c05ce72f629cd8ec29deff1ed97d5) @bang9